acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/condition/priv/EeaSendRawTransactionSuccess.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance.dsl.condition.priv;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.tests.acceptance.dsl.condition.Condition;
+import org.hyperledger.besu.tests.acceptance.dsl.node.Node;
+import org.hyperledger.besu.tests.acceptance.dsl.transaction.privacy.EeaSendRawTransactionTransaction;
+
+import java.util.concurrent.CompletableFuture;
+
+public class EeaSendRawTransactionSuccess implements Condition {
+
+  private final EeaSendRawTransactionTransaction sendRawTransactionTransaction;
+  private final CompletableFuture<Hash> completeableFuture;
+
+  public EeaSendRawTransactionSuccess(
+      final EeaSendRawTransactionTransaction sendRawTransactionTransaction,
+      final CompletableFuture<Hash> completableFuture) {
+    this.sendRawTransactionTransaction = sendRawTransactionTransaction;
+    this.completeableFuture = completableFuture;
```

Comment:
Why do we need a completableFuture here? Generally the ATs are written in synchronous way so that we perform some operation using node.execute and then node.verify to wait for the condition to succeed.

Reply:
Incorrect fix for a test setup issue.  Removed.